### ISSUE TITLE
Fizy Service Request

### ISSUE THREAD

**Issue by GAppsGames** [PreMiD/PreMiD#112]
**What's the name of the service?**
Fizy

**What should the Presence display?**
Information about listening music.

**Why do you want support for this service?**
This service is widely used in Turkey. (Google Play **10.000.000+** Downloads ( https://play.google.com/store/apps/details?id=com.turkcell.gncplay&hl=tr ) | App Store **I not found this :(** Downloads ( https://itunes.apple.com/tr/app/fizy-müzik-video/id404239912?l=tr&mt=8 ) )

**Are you able to provide an image for the service?**
Yes, this https://media.kktcell.com/awh/287/296//Files/shop/content/services/fizy-logo1.jpg

**Comment by GAppsGames** [comment]
> **What's the name of the service?**
> Fizy
> 
> **What should the Presence display?**
> Information about listening music.
> 
> **Why do you want support for this service?**
> This service is widely used in Turkey. (Google Play **10.000.000+** Downloads ( https://play.google.com/store/apps/details?id=com.turkcell.gncplay&hl=tr ) | App Store **I not found this :(** Downloads ( [https://itunes.apple.com/tr/app/fizy-müzik-video/id404239912?l=tr&mt=8](https://itunes.apple.com/tr/app/fizy-m%C3%BCzik-video/id404239912?l=tr&mt=8) ) )
> 
> **Are you able to provide an image for the service?**
> Yes, this https://media.kktcell.com/awh/287/296//Files/shop/content/services/fizy-logo1.jpg

**Comment by DooML** [comment]
Does it have a web player / available without downloading any apps? because if it's not, then that would not be possible.

**Comment by GAppsGames** [comment]
> Does it have a web player / available without downloading any apps? because if it's not, then that would not be possible.

Yes, have a web player. play.fizy.com

**Comment by Fruxh** [comment]
@GAppsGames But you know that we can't detect the songs from your phone, right?

**Comment by GAppsGames** [comment]
@Fruxh yes i know. Web Player is this : https://redstheunbeaten.is-inside.me/ODyASFHC.png
